### PR TITLE
Update Postgres configuration on Travis to avoid error.

### PR DESCRIPTION
**Pull request by mbcosta**
Update Postgres configuration on Travis to avoid error.

cc @renatonlima @rvalyi

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You can take the occasion to update whole file with one less build and removing old Transifex stuff

**Comment by pedrobaeza** [comment]
Why putting 9.2 instead of 9.6?

**Comment by mbcosta** [comment]
Hi @pedrobaeza , I receive the same error with 9.6, I just going to check if it's a version problem ( looking in other repo projects in OCA at version 10 ) or do you think it's is not relation ?

**Comment by danimaribeiro** [comment]
You can use:
```
services:
  - postgresql
```

**Comment by rvalyi** [comment]
@mbcosta you made a mistake, look twice at the master template:
https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L13
you put postgres inside the ```apt``` section but it's inside the ```addons``` parent section instead (it's a service, not an apt package here).

File: .travis.yml [hunk 1]
```diff
@@ -16,9 +17,7 @@ addons:
 
 env:
   global:
-  - VERSION="10.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
-  - TRANSIFEX_USER='transbot@odoo-community.org'
-  - secure: "FmJknQWKDSXzrQZm5+PNNhQwEJzA4IVQpT6UASSthvQA3i88nsX3khsHBRAoQhJr7NTinIp4nIvsvVvsrKXSnbDNY0bScrx8dCDD2weKZnsrD4Khh6Gn3pFwCrwJiPccnMUnFuBxbJzV/tZRrupQIcIfFg8AApCYJ4Qm+nb5XuA="
+  - VERSION="10.0" TESTS="0" LINT_CHECK="0"
   matrix:
   - LINT_CHECK="1"
   - TRANSIFEX="1"
```
Comment (pedrobaeza) [inline .travis.yml]: Remove this one and add `MAKEPOT="1"` in the last build.

**Comment by mbcosta** [comment]
Thanks all for help

**Pull request merged**